### PR TITLE
Fixed errors when trying to set API key.

### LINKED ISSUE #7: WakaTime throws an error when trying to enter API Key after initial config
Noticing that attempting to run the `api key` command to check the API key throws this error:
Screen Shot 2021-03-23 at 3 53 32 PM

My `.wakatime.cfg` file has:
```
[settings]
api_key = <key>
```

Nova version: 5 Build 245280
macOS: 11.2.3 (20D91)
Device: MacBook Air (M1, 2020)

### PR DESCRIPTION
**Pull request by mikebronner**
This PR attempts to address the errors thrown by the extension when trying to set the API key.

Fixes #7

### CODE REVIEW

File: Scripts/main.js [hunk 1]
```diff
@@ -192,11 +192,12 @@ function cliPath() {
...
 
 function getSetting(section, key) {
-  try {
-    const config = nova.fs.open(getConfigFile(), 'r', 'utf-8');
-  } catch (e) {
-    return '';
+  const config = nova.fs.open(getConfigFile(), 'r', 'utf-8');
+
+  if (! config) {
+      return;
```
Comment (alanhamlett) [inline Scripts/main.js:198]: We should return `''` empty string.

File: Scripts/main.js [hunk 2]
```diff
@@ -192,11 +192,12 @@ function cliPath() {
 }
 
 function getSetting(section, key) {
-  try {
```
Comment (alanhamlett) [inline Scripts/main.js:195]: The docs don't mention [open](https://docs.nova.app/api-reference/file-system/#openpath-mode-encoding) throwing an error, but it doesn't hurt to leave this exception handling here.

**Comment by mikebronner** [comment]
Hi @alanhamlett, please feel free to make any adjustments you deem necessary to the PR. I left the setting enabled that allows owners to make changes. :)

**Comment by alanhamlett** [comment]
Working from your changes, I've pushed e0111d58e1c8d4223e239bce04097f8821a881b7 and released the fix in v2.0.2. Can you see if that works for you?

**Pull request closed without merging**